packaging/pre_build_script_windows.sh

```diff
@@ -1,12 +1,18 @@
-python -m pip install -U numpy packaging pyyaml setuptools wheel
+set -eou pipefail
 
-# Install TRT from PyPi
-TRT_VERSION=$(${CONDA_RUN} python -c "import yaml; print(yaml.safe_load(open('dev_dep_versions.yml', 'r'))['__tensorrt_version__'])")
+pip install -U numpy packaging pyyaml setuptools wheel
 
-python -m pip install tensorrt==${TRT_VERSION} tensorrt-${CU_VERSION::4}==${TRT_VERSION} tensorrt-${CU_VERSION::4}-bindings==${TRT_VERSION} tensorrt-${CU_VERSION::4}-libs==${TRT_VERSION} --extra-index-url https://pypi.nvidia.com
+# Install TRT from PyPI
+TRT_VERSION=$(python -c "import yaml; print(yaml.safe_load(open('dev_dep_versions.yml', 'r'))['__tensorrt_version__'])")
+pip install tensorrt==${TRT_VERSION} tensorrt-cu12-bindings==${TRT_VERSION} tensorrt-cu12-libs==${TRT_VERSION} --extra-index-url https://pypi.nvidia.com
```

Comment:
Could the specialization on CUDA version in the `tensorrt` package names be added back for the `bindings` and `libs` packages?

Reply:
Fixed